chapters/12-project-management.Rmd

```diff
@@ -1,34 +1,55 @@
 - Milestones
 
-
-
-## Final exercise
-
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
-
-1. Create a [project roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). 
-- Create a roadmap for your project describing short-term, medium-term, and long-term project goals. This can be a GitHub issue or a file called ROADMAP.md. Example: [STEMMM Role Models roadmap](https://github.com/KirstieJane/STEMMRoleModels/issues/1).
-
-2. Issue tracking and labeling 
-- Create several GitHub issues for your project using your roadmap for inspiration. These can range from simple project improvements (like scanning for typos in the README) to complex feature additions or bugs to be fixed. 
-- Create several issue labels that make sense for your project, i.e. good first bug, discussion, help wanted, documentation, feature, etc. Label your issues with one or more appropriate labels.
-- Add a link to one or more issue categories to CONTRIBUTING.md. Add a link to issues that a new contributor could attempt, i.e. "good first bug".
-- Add links to your issues and/or issue labels to your roadmap.
-
-3. GitHub milestones and Projects
-- Look at some examples of packages that use GitHub milestones and GitHub Projects. Are there situations in which one project management approach is better than the other? Which do you prefer, or which do you feel suits your project better?
+[project-roadmap]: https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/
+
+<!-- TODO: This is better here, not in the final exercise. -->
+<!-- Example: [STEMMM Role Models -->
+<!-- roadmap](https://github.com/KirstieJane/STEMMRoleModels/issues/1). -->
+
+## Final exercise {#12-final-exercise}
+
+Complete these tasks as part of the package for your first project assignment.
+Refer to the project assignment TODO: Add ref to assignment here?
+for more details. As with the other final exercises,
+we will use the `kenyaweather` package as an example and reference.
+Don't forget to add and commit any changes you've made into the Git history
+after each task, and pushing to GitHub. Make use of the usethis `pr_*` workflow
+described in Chapter \@ref(github-workflows).
+
+1. Create a [roadmap][project-roadmap] for your project describing short-term,
+medium-term, and long-term project goals. For now keep these goals about completing
+the first project assignment. This can be a GitHub issue or a file called
+`ROADMAP.md`. TODO: We should decide on one, to restrict learners choice.
+
+2. Using your roadmap as inspiration for todo items, create new Issues for a few
+of your project's tasks.
+    - Create several new Issue Labels. GitHub provides a few good default ones.
```

Comment:
Let's recommend a set of labels, I think `bug`, `enhancement`, `documentation` are a good basis and then some project-specific labels can be added, e.g. indicating probalems with upstreams packages etc.

Should we lso teach issue templates? I made these that I like https://github.com/altair-viz/altair/issues/new/choose

Reply:
Great idea!